### DISCUSSION TITLE
Add support for English, Sweden, Norway. France, and Norwegian regions

### DISCUSSION THREAD

**Discussion by pandafliesinspace** [holypower777/global-chat#4] [Ideas]
As a USER I want to see my messages that I've sent to twitch channels with UK, Sweden, Norway, France, and Norwegian tags

**Comment by holypower777** [comment]
I would be happy to support these regions, but for this I do not have extra money in order to maintain the database. So you can [support](https://boosty.to/holypower77) me if you want.
